distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -182,10 +182,14 @@ grant codeBase "file:/org.apache.karaf.deployer.blueprint/org.apache.karaf.deplo
 }
 
 grant codeBase "file:/org.apache.karaf.jaas.modules" {
-    permission java.io.FilePermission "${ddf.home.perm}etc${/}users.properties", "read";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}users.properties", "read,write";
```

Comment:
❓ Are those required because they are on the stack? If so, could the `admin-core-insecuredefaults` simply extends its privileges when it is performing the deletion?

Reply:
In DDF, we're still operating under the status of adding permissions to bundles that are on the stack. When that changes, this could be revisited.

Reply:
Not necessarily. If your bundle is performing something that requires permissions then in some case this might simply require giving your bundle the permissions and extending your privileges. We shouldn't bleed implementation if we don't have to.

Reply:
Actually, when I look closer, this most likely is directly required for the second half of this PR which is making the jaas commands executable.

Reply:
Those permissions are added to get the jaas commands working